packages/react-ui/src/app/features/builder/builder-hooks.ts

```diff
@@ -597,6 +597,17 @@ const applyMidpanelAction = (state: BuilderState, action: MidpanelAction) => {
   };
 };
 
+async function deleteChatRequest(flowVersion: FlowVersion, stepName: string) {
```

Comment:
I extracted this function, I think it's cleaner for it to be outside `updateFlowVersion`